doc/source/conf.py

```diff
@@ -183,7 +183,7 @@
 scv_banner_greatest_tag = True
```

Comment:
That does not solve it because it does *not* show the banner in master. That's where it should be. :/